tests/test_dcos_e2e/docker_utils/test_loopback.py

```diff
@@ -18,21 +18,34 @@ def test_loopback(self) -> None:
         client = docker.from_env(version='auto')
 
         with DockerLoopbackVolume(size=1) as device:
```

Comment:
This makes me think - what do you think of changing this parameter name to be something like `size_megabytes`?

Reply:
The best solution would be a `Size` type, but I'm coming from statically typed languages, not sure if this is a common pattern in Python.

Reply:
The most common thing is to use something like `size_megabytes`. However, I have used [bitmath](https://pypi.org/project/bitmath/) to provide a type for sizes.

Reply:
I'll leave it to you to choose what to do, where doing nothing is an acceptable (to me) but not preferred choice. In a follow-up I hope so we can ship this.

Reply:
`size_megabytes` it is. Will create an extra patch for that.